app/src/main/java/com/garethevans/church/opensongtablet/Metronome.java

```diff
@@ -225,34 +227,25 @@ static void startstopMetronome(Activity activity, Context c, boolean showvisual,
             // Start the metronome
             StaticVariables.metronomeonoff = "on";
             StaticVariables.whichbeat = "b";
+            // This is a state indicator set in this function only
+            StaticVariables.clickedOnMetronomeStart = true;
             metroTask = new MetronomeAsyncTask(pan,vol,barlength);
             try {
                 metroTask.executeOnExecutor(AsyncTask.THREAD_POOL_EXECUTOR);
             } catch (Exception e) {
                 Log.d("d","Error starting the metronome");
             }
             startstopVisualMetronome(showvisual,metronomeColor);
-
-        } else if (checkMetronomeValid(c) && StaticVariables.metronomeonoff.equals("on")) {
+        // IV - A stop perhaps does not need to consider if it is valid
+        } else if (StaticVariables.metronomeonoff.equals("on")) {
```

Comment:
Going to need to look at this further.  Not sure if it is this change or another, but now when a song is playing a metronome, swiping to the next song does not display it and the metronome keeps playing.
Original action was to display a warning that it wasn't set, but the next song did load

Reply:
Happy not to open the popup (just display toast that metronome is not set), then stop the metronome (currently in my logic it wrongly keeps playing)

Reply:
OK